src/components/header/renderer.tsx

```diff
@@ -242,6 +249,28 @@ class Header extends React.Component<Props, State> {
     dlButton.classList.remove('disabled');
   }
 
```

Comment:
No need to be async

Reply:
Yes, but we don't need it for json.

Reply:
Yes , I got it and pushed the changes too , this was a previous fork so there are some linting issues , I will fix them in my next pr.